crates/astria-conductor/src/block_verifier.rs

```diff
@@ -45,23 +45,55 @@ use tracing::{
     warn,
 };
 
+#[async_trait::async_trait]
+pub(crate) trait SequencerClient {
+    async fn validators(
```

Comment:
Is this functionality that should live in conductor only, or should we extend the `astria-sequencer-client`'s extension trait?

Reply:
we can - either one is fine imo. currently, we only use this here though